### ISSUE TITLE
[Bug]: Typo in error message when an external function is declared with no java class specified

### ISSUE THREAD

**Issue by lochana-chathura** [ballerina-platform/ballerina-lang#41620]
### Description

$subject. Run the following code.
```bal
function foo() {
    function (typedesc<int> x) returns int fn1 = bar;
}

function bar(typedesc<int> y) returns y = external;
```

It will give `error: cannot find full qualified class name for extern function : lochanaj/Sample/0/bar`

Note the typo `extern` in the error message.

### Affected Version(s)

2201.8.1

**Comment by MaryamZi** [comment]
`external` functions used to be `extern` functions at one point :) Looks like we've missed updating the error here.

**Comment by gabilang** [comment]
What is the correct word to use here? `external function` or `foreign function`

**Comment by lochana-chathura** [comment]
> What would be the correct word to use here? `external function` or `foreign function`

external function, as per the spec terms and syntax.

**Comment by gabilang** [comment]
The above requested fix is no longer required since the old-style external function wrapper related implementations have been removed from the `ExternalMethodGen.java` class. Hence closing this issue.

**Comment by lochana-chathura** [comment]
New issue to be addressed: #41744
